src/styles/partials/components/_summary-list.scss

```diff
@@ -0,0 +1,20 @@
+
+.dg_summary-list__item {
+  display: flex;
+  flex-wrap: no-wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  background-color: $gray-lightest;
+  color: $gray;
+  padding: $margin-large;
```

Comment:
this should be `margin-bottom: $default-margin`. this might change when design team sees it, but this is what matches in xd.

Reply:
sorry should have clarified, padding should be replaced with margin-bottom. So you no longer need line 7.

Reply:
done